u8timeseries/backtesting/backtesting.py

```diff
@@ -259,3 +252,117 @@ def plot_residuals_analysis(residuals: TimeSeries, num_bins: int = 20):
     ax3.set_ylabel('ACF value')
     ax3.set_xlabel('lag')
     ax3.set_title('ACF')
+
+
+def backtest_gridsearch(model_class: type, parameters: dict, 
+                        train_series: TimeSeries, 
+                        val_series: Optional[TimeSeries] = None,  # only used by split mode
+                        fcast_horizon_n: int = 5, num_predictions: int = 10,  # only used by expanding window mode
```

Comment:
I think all the parameters should be on a separate line

Reply:
done